package/MDAnalysis/coordinates/GRO.py

```diff
@@ -377,7 +387,7 @@ def write(self, obj):
             # all attributes could be infinite cycles!
             for atom_index, resid, resname, name in zip(
                     range(ag_or_ts.n_atoms), resids, resnames, names):
-                truncated_atom_index = util.ltruncate_int(atom_index + 1, 5)
+                truncated_atom_index = util.ltruncate_int(atom_indexs[atom_index], 5)
```

Comment:
I'd put `atom_indexs` into the `zip` call to make it similar to `resid` etc

Reply:
@richardjgowers atom_index is used Later in the code as an index
pos=positions[atom_index]
vel=velocities[atom_index]

Thus, if I zip the atom_indexs in to the zip, I will need to create another index start from 0.